bites.py

```diff
@@ -145,14 +155,14 @@ def take_all_turns(self):
         self.render_game()
         if None not in self.anthill: return
 
-  def calculate_and_print_scores(self):
+  def calculate_and_print_scores(self, standard_tokens_for_trail):
```

Comment:
When `calculate_and_print_scores()` is being called in `play_full_game()`, `standard_tokens_for_trail` is not being passed. This will cause problems. Consider making `standard_tokens_for_trail` an attribute within the `__init__` method, so that it can be accessed in here as `self.standard_tokens_for_trail` instead of being passed in as an argument.